swig/openscap_api.py

```diff
@@ -206,6 +238,30 @@ def __getter_wrapper(*args, **kwargs):
 
         return __getter_wrapper
 
+    def introspect_all(self):
+        """ Listing all builtin functions accessible through SWIG """
+        return OSCAP.__dict__
+
+    def introspect(self):
```

Comment:
If you have `introspect_constants` and `introspect_all`, would it make sense to rename `introspect` to `introspect_functions`?